DQMOffline/HLTScouting/python/HLTScoutingPostProcessing_cff.py

```diff
@@ -0,0 +1,19 @@
+'''
+Scouting DQM sequences for offline DQM developed for 2025 pp data-taking
+and used by DQM GUI (DQMOffline/Configuration):
+currently running EGM and MUO monitoring modules.
+'''
+
+import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
+from DQMOffline.HLTScouting.HLTScoutingDqmOffline_cff import *
+
+from HLTriggerOffline.Scouting.ScoutingMuonMonitoring_Client_cff import *
+from HLTriggerOffline.Scouting.HLTScoutingEGammaPostProcessing_cff import *
+
+hltScoutingMuonPostProcessing = cms.Sequence(muonEfficiencyNoVtx
+                                             * muonEfficiencyVtx  
+                                             * muonTriggerEfficiency
+                                             )
+
+hLTScoutingPostProcessing = cms.Sequence(hltScoutingMuonPostProcessing + hltScoutingEGammaPostProcessing)
```

Comment:
```suggestion
HLTScoutingPostProcessing = cms.Sequence(hltScoutingMuonPostProcessing + hltScoutingEGammaPostProcessing)
```

should fix the crash.

Reply:
Bug noted, thank you! It should be fixed in [Implementation of suggested corrections in PR47235 (config files fix)](https://github.com/cms-sw/cmssw/pull/47235/commits/37cd45cb68636d6f5acd40c4ef5b40e364f1bd08).